databricks-kube/src/context.rs

```diff
@@ -19,28 +19,59 @@ pub struct Context {
     api_secret_store: Arc<Store<Secret>>,
 }
 
-#[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema)]
+#[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema, Debug)]
 pub struct DatabricksAPISecret {
     pub databricks_url: String,
     pub access_token: String,
 }
 
+impl Default for DatabricksAPISecret {
+    fn default() -> Self {
+        Self {
+            databricks_url: String::from("default_url"),
+            access_token: String::from("default_token"),
+        }
+    }
+}
+
 #[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema)]
 pub struct OperatorConfiguration {
     pub api_secret_name: String,
+    pub default_poll_interval: String,
+    pub default_timeout_seconds: String,
+    pub default_requeue_interval: String,
+    pub secrets: Option<BTreeMap<String, String>>
```

Comment:
I think it may be better to make this `Option<DatabricksAPISecret>`. But it's probably best to remove from this struct (more in comments around `Context`)